docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -1,79 +1,53 @@
 ---
 title: Moderated Sessions
-description: Moderated Sessions
-h1: Moderated Sessions
+description: Describes the purpose of moderated sessions and how to configure roles to support moderated sessions in a Teleport cluster.
+keywords:
+  - SSH
+  - Kubernetes
+  - session moderation
+  - audit
 ---
 
-## Introduction
+Moderated sessions allow you to define requirements for other users to be present 
+in an active server or Kubernetes session started by another user. Depending on the 
+requirements you specify, users who are allowed to join other users' sessions can be 
+granted permission to do the following:
 
-Moderated Sessions allows Teleport administrators to define requirements for
-other users to be present in a server or Kubernetes session. Depending on the
-requirements, these users can observe the session in real time, participate in
-the session, and terminate the session at will.
+- Observe another user's session in real time.
+- Participate interactively in another user's session.
+- Terminate another user's session at will.
 
-In addition, Teleport administrators can [define rules](#join_sessions) that allow users to join each other's
-sessions from `tsh` and the Web UI.
+The most common use cases for moderated sessions involve the following scenarios:
 
-<Notice type="warning">
-
-  Moderated Sessions requires Teleport Enterprise or Teleport Enterprise Cloud.
-
-</Notice>
-
-### Use cases
-
-Moderated Sessions are useful in the following scenarios:
-- When you have stringent security requirements and need to have people watching
+- You have strict security or compliance requirements and need to have people watching
   over user-initiated sessions on a set of servers.
-- When you want to share a terminal with someone else to be able to instruct or
-  collaborate.
-
-## Policies
-
-Moderated Sessions makes use of RBAC policies to allow for fine grained control
-over who can join a session and who is required to be present to start one.
-
-The system is based around **require policies** and **allow policies**.
-
-Require policies define a set of conditions that must be a met for a session to
-start or run. A minimum of one policy from each relevant role the user has must
-match for the session to start.
-
-Allow policies are used to define what sessions a user can join and under what
-conditions they may join a session.
-
-## Configuring Moderated Sessions
-
-### `require_session_join`
+- You want to share a terminal with someone else to be able to instruct or collaborate.
+- You need the ability to pause or terminate active sessions.
 
-#### Options
+Note that you can share terminal sessions using any Teleport edition. However, you must 
+have Teleport Enterprise or Teleport Enterprise Cloud to join a session as a **moderator** 
+with permission to terminate sessions.
```

Comment:
Another way to phrase this would be to say that defining a `require_session_join` policy requires enterprise. If you try to define a role with this in OSS you will get an error.